backend/LexBoxApi/Auth/LexAuthService.cs

```diff
@@ -16,15 +17,18 @@ namespace LexBoxApi.Auth;
 
 public class LexAuthService
 {
+    public const string RefreshHeaderName = "lexbox-refresh";
```

Comment:
Or did you make it vague on purpose?
```suggestion
    public const string RefreshHeaderName = "lexbox-refresh-user";
```

Reply:
Haha, no I did want a better name but wasn't sure what to do. What about `refesh-user`? Or... make the value `jwt` and leave the vague name. IDK if we'd ever need to refresh something else though. gql can do that fine on it's own so maybe that's overkill.

Reply:
I like `lexbox-refresh-jwt`. Without `lexbox` it'd be fairly mysterious if you just happened to notice it somewhere.